framework/decode/vulkan_pre_process_consumer.h

```diff
@@ -0,0 +1,774 @@
+/*
+** Copyright (c) 2025 LunarG, Inc.
+**
+** Permission is hereby granted, free of charge, to any person obtaining a
+** copy of this software and associated documentation files (the "Software"),
+** to deal in the Software without restriction, including without limitation
+** the rights to use, copy, modify, merge, publish, distribute, sublicense,
+** and/or sell copies of the Software, and to permit persons to whom the
+** Software is furnished to do so, subject to the following conditions:
+**
+** The above copyright notice and this permission notice shall be included in
+** all copies or substantial portions of the Software.
+**
+** THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+** IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+** FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+** AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+** LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+** FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+** DEALINGS IN THE SOFTWARE.
+*/
+
+#ifndef GFXRECON_DECODE_VULKAN_PRE_PROCESS_CONSUMER_H
+#define GFXRECON_DECODE_VULKAN_PRE_PROCESS_CONSUMER_H
+
+#include "generated/generated_vulkan_consumer.h"
+
+GFXRECON_BEGIN_NAMESPACE(gfxrecon)
+GFXRECON_BEGIN_NAMESPACE(decode)
+
+const std::string DUMP_ARG_BEGIN_COMMAND_BUFFER = "BeginCommandBuffer";
+const std::string DUMP_ARG_DRAW                 = "Draw";
+const std::string DUMP_ARG_RENDER_PASS          = "RenderPass";
+const std::string DUMP_ARG_BEGIN_RENDE_PASS     = "BeginRenderPass";
```

Comment:
Why not DUMP_ARG_BEGIN_RENDER_PASS?

Reply:
Thank you. Nice catch. It was typo. Fixed.